p2p-specs/p2p-interface.md

```diff
@@ -405,7 +412,8 @@ The response MUST consist of a single `response_chunk`.
 
 Clients SHOULD immediately disconnect from one another following the handshake above under the following conditions:
 
-1. If the `supported_mempools` and client's own list of supported mempools are disjoint.
+1. If `chain_id` does not match the node's local `chain_id`, since the peer is on another network.
```

Comment:
wrong chainid is a configuration error, either of this bundler or the other: the mempool is for a single chain.
block-number/hash are to check the liveliness status of the peer node, and reorg status.